content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -284,6 +284,15 @@ If your network is not working or CoreDNS is not in the `Running` state, check o
 [troubleshooting guide](/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/)
 for `kubeadm`.
 
+### Managed node labels
+
+By default, kubeadm enables the [NodeRestriction](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
+admission controller that restricts what labels can be self-applied by kubelets on node registration.
+The admission controller documentation covers what labels are permitted to be used with the kubelet `--node-labels` option.
+The `node-role.kubernetes.io/control-plane` label is such a restricted label and kubeadm manually applies it using
```

Comment:
https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction _does not_ list
`node-role.kubernetes.io/control-plane` as a restricted label @PriyanshuAhlawat 

It is however not on the allowlist, and is a **reserved** label by dint of not being on the allowlist.

Reply:
Should I remove `node-role.kubernetes.io/control-plane` from the pr?

Reply:
The relevant part from the link is:
> Use of any other labels under the kubernetes.io or k8s.io prefixes by kubelets is reserved, and may be disallowed or allowed by the NodeRestriction admission plugin in the future.

As such, node-role labels are restricted.
Kubectl and kubeadm are the only relevant users of these. Also core k8s has no concept of a node-role.